vistrails/gui/uvcdat/diagnosticsDockWidget.py

```diff
@@ -852,7 +852,10 @@ def displayCell(self, res30, row, column, sheet="Sheet 1"):
          from packages.uvcdat_cdms.init import get_canvas, get_gm_attributes, original_gm_attributes
          from gui.uvcdat.uvcdatCommons import gmInfos
          Gtype = res30.type
-         G = V[Gtype]
+         if res30.type=="Taylor":
+             G=V["Taylordiagram"]
```

Comment:
Is `V` a `variable`? `G` something graphics related? `pvars` short for `plot_variables`?